### PR TITLE
Respect language settings when geting a list of commands

### PR DESCRIPTION
**Pull request by frenzymadness**
Fixes: https://github.com/tldr-pages/tldr-python-client/issues/238

If you agree with this approach, I can add tests for that.

### CODE REVIEW

File: tldr.py [hunk 1]
```diff
@@ -305,17 +305,24 @@ def get_page(
...
-            path = get_cache_dir() / 'pages' / platform
-            if not path.exists():
-                continue
-            commands += [file.stem for file in path.iterdir() if file.suffix == '.md']
+            for language in languages:
+                pages_dir = f'pages.{language}' if language != 'en' else 'pages'
+                path = get_cache_dir() / pages_dir / platform
+                if not path.exists():
+                    continue
+                commands += [file.stem for file in path.iterdir() if file.suffix == '.md']
```
Comment (vitorhcl) [inline tldr.py:325]:
```suggestion
                commands += [file.stem for file in path.iterdir() if file.suffix == '.md']
                break
```

We can exit the language loop when a page for a given language is found, right? I think we only need to continue to search for pages for other platforms.

**Review by vitorhcl — COMMENTED** [review]
Thank you! I have a suggestion.

File: tldr.py (hunk 1 above)
Comment (vitorhcl) [inline tldr.py:325]:
```suggestion
                commands += ["file.stem ({language})" for file in path.iterdir() if file.suffix == '.md']
```

If we do want to search for pages in all languages, we should show the language of each page.

Comment (frenzymadness) [reply tldr.py:325]: Good idea but your suggestion does not make the string an f-string. Could you please update it?

Comment (kbdharun) [reply tldr.py:325]: @frenzymadness The suggestion has been updated (feel free to apply it).

**Comment by frenzymadness** [comment]
I've added a test and squashed all fixups into a single commit.

**Review by vitorhcl — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**